.github/workflows/test.yml

```diff
@@ -16,7 +16,7 @@ jobs:
         with:
           name: artifact_cppcheck
           path: ./artifact_cppcheck.txt
-      - run: g++ -std=c++17 *.cpp -Wall -Wextra -g -fsanitize=address
+      - run: g++ -std=c++17 *.cpp -Wall -Wextra -g
```

Comment:
Ok, így már a valgrind lefut, de az igazán jó megoldás az lenne, ha azzal is meg ASannal is lefutna, mert van egy kis eltérés abban, hogy milyen hibákat tudnak detektálni. De ez nem volt kérés a feladatban, és látom, hogy mindkét módszert tudod használni, úgyhogy maradhat.